bdns/dns.go

```diff
@@ -373,18 +374,28 @@ func (dnsResolver *DNSResolverImpl) LookupHost(ctx context.Context, hostname str
 
 	var addrs []net.IP
 
+	// First append any IPv6 addresses from the AAAA records since we prefer these
```

Comment:
Now that we do the v4/v6 split in the VA, we no longer need to specifically order the records. I think you can revert this whole file, along with its test.